examples/angular/row-selection/src/app/app.component.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  signal,
+  TemplateRef,
+  viewChild,
+} from '@angular/core'
+import {
+  ColumnDef,
+  createAngularTable,
+  FlexRenderComponent,
+  FlexRenderDirective,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  RowSelectionState,
+} from '@tanstack/angular-table'
+import { FilterComponent } from './filter'
+import { makeData, type Person } from './makeData'
+import { FormsModule } from '@angular/forms'
+import {
+  TableHeadSelectionComponent,
+  TableRowSelectionComponent,
+} from './selection-column.component'
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [FilterComponent, FlexRenderDirective, FormsModule],
+  templateUrl: './app.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppComponent {
+  private readonly rowSelection = signal<RowSelectionState>({})
+  readonly globalFilter = signal<string>('')
+  readonly data = signal(makeData(10_000))
+
+  readonly ageHeaderCell =
+    viewChild.required<TemplateRef<unknown>>('ageHeaderCell')
+
+  readonly columns: ColumnDef<Person>[] = [
+    {
+      id: 'select',
+      header: () => {
```

Comment:
@riccardoperra What would be the new flex render way to do this? It's no longer exported. 

<img width="587" alt="image" src="https://github.com/TanStack/table/assets/28243511/2b39120c-85e8-4afb-95fa-f42be76eed89">

Reply:
I think this is a cache issue since `FlexRenderComponent` is exported 🤔 or you have to rebuild the package https://github.com/TanStack/table/blob/67a0b105e047c95b16f781b55a8eea8818b367c0/packages/angular-table/src/index.ts#L25

Reply:
you're right, my mistake